src/reference/asciidoc/whats-new.adoc

```diff
@@ -106,10 +106,15 @@ See "`<<jdbc>>`" for more information.
 === FTP and SFTP Changes
 
 A `RotatingServerAdvice` is now available to poll multiple servers and directories with the inbound channel adapters.
-See "`<<ftp-rotating-server-advice>>`" and "`<<sftp-rotating-server-advice>>`" for more information.
+See "<<ftp-rotating-server-advice>>" and "<<sftp-rotating-server-advice>>" for more information.
 
 Also, inbound adapter `localFilenameExpression` instances can contain the `#remoteDirectory` variable, which contains the remote directory being polled.
 The generic type of the comparators, used to sort the fetched file list for the streaming adapters, has changed from `Comparator<AbstractFileInfo<F>>` to simply `Comparator<F>`.
 See <<ftp-streaming>> and <<sftp-streaming>> for more information.
 
 In addition, the synchronizers for inbound channel adapters can now be provided with a `Comparator`; this is useful when using `maxFetchSize` to limit the files retrieved.
+
+==== Twitter Support
+
+Since Spring Social project is in the https://spring.io/blog/2018/07/03/spring-social-end-of-life-announcement[End of Life], the Twitter support in Spring Integration has been moved to the Extensions project.
```

Comment:
Since Spring Social project **moved to** https://spring.io/blog/2018/07/03/spring-social-end-of-life-announcement[End of Life **Status**], Twitter support in Spring Integration has been moved to the Extensions project. 

Add URL to the extension.